shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -566,11 +567,12 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
         out.write(buf, 0, t);
         t = is.read(buf);
       }
-      if (!tmpDst.renameTo(dstFile)) {
+      File outputFile = new File(dstFile.getAbsolutePath() + "/" + fileName));
```

Comment:
dstFile is just the path not the file. If users just set the dstfolder, dstFile would be the path not the file.